src/main/java/Task_32/models/City.java

```diff
@@ -0,0 +1,57 @@
+package Task_32.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table (name = "city")
+public class City {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column (name = "name")
+    private String name;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "student_id")
+    private Student student;
```

Comment:
This foreign key and the relation is incorrect.
It should be swapped - ONE City for the ONE Student, but MANY Students for the ONE City.
Same is for Group